packages/generator/src/edmx-to-vdm/common/action-function-return-types.ts

```diff
@@ -0,0 +1,139 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+import {
+  VdmActionImportReturnType,
+  VdmComplexType,
+  VdmEntity,
+  VdmFunctionImportReturnType,
+  VdmReturnTypeCategory
+} from '../../vdm-types';
+import {
+  isCollectionType,
+  parseTypeName,
+  propertyJsType
+} from '../edmx-to-vdm-util';
+
+export function parseFunctionImportReturnTypes(
+  returnType: string | undefined,
+  entities: VdmEntity[],
+  complexTypes: Omit<VdmComplexType, 'factoryName'>[]
+): VdmFunctionImportReturnType {
+  return parseReturnTypes(
+    returnType,
+    entities,
+    complexTypes
+  ) as VdmFunctionImportReturnType;
+}
+
+export function parseActionImportReturnTypes(
+  returnType: string | undefined,
+  entities: VdmEntity[],
+  complexTypes: Omit<VdmComplexType, 'factoryName'>[]
+): VdmActionImportReturnType {
+  return parseReturnTypes(
+    returnType,
+    entities,
+    complexTypes
+  ) as VdmActionImportReturnType;
+}
+
+function parseReturnTypes(
```

Comment:
Same as for the parameters, why not just exporting this function instead of the two above. Does anyone down the road even care about this being returned `as VdmActionImportReturnType` (or VdmActionImportReturnType)?